src/editor fullscreen.js

```diff
@@ -0,0 +1,94 @@
+/**
+🖥️
+@file editor fullscreen
+@summary Add a ui button and hotkey to the editor to toggle fullscreen while playtesting the game.
+@license MIT
+@author Violgamba (Jon Heard)
+
+@description
+Add a ui button and hotkey to the editor to toggle fullscreen mode while playtesting the game.
+If 'gamepad-input' plugin is used, a gamepad's 'start' button triggers the 'enter' key.  This is
+why 'enter' is the default fullscreen hotkey, to allow toggling fullscreen from a gamepad.
+
+
+HOW TO USE:
+1. Import this plugin into your game.
+2. Run the game.
+3. Press the 'enter' key.  Note that the game becomes fullscreen.
+4. Press the 'enter' key again.  Note that the game becomes non-fullscreen.
+5. Find the fullscreen button on the lower left of the ui (while the game is running).
+6. Press the fullscreen button.  Note that the game becomes fullscreen.
+NOTE: if this plugin's configuration is changed, the key to press may not be 'enter'.
+
+
+// Which keyboard key triggers fullscreen
+//!CONFIG fullscreen-hotkey (text) "Enter"
+*/
+
+//! CODE_EDITOR
+
+function setupFullscreenToggle() {
+	const KEY_FOR_FULLSCREEN = FIELD(CONFIG, 'fullscreen-keycode', 'text') || 'Enter';
+
+	// Fullscreen toggle logic
+	window.EDITOR.toggleFullscreen = function toggleFullscreen() {
+		const canvas = this.playtestIframe.contentWindow.document.getElementById('player-canvas');
+		if (!canvas) {
+			return;
+		}
```

Comment:
nit: you can write this more concisely like:
```suggestion
		if (!canvas) return;
```